corehq/apps/accounting/bootstrap/utils.py

```diff
@@ -184,14 +184,16 @@ def _ensure_software_plan(plan_key, product, product_rate, verbose, apps):
 
 
 def _software_plan_name(plan_key, product, product_rate):
-    plan_name = (
-        (f"{product_rate.name} Trial" if plan_key.is_trial else f"{product_rate.name} Edition")
-        if product is None else product.name)
-    if plan_key.is_annual_plan:
-        plan_name = f"{plan_name} - Pay Annually"
-    if plan_key.is_report_builder_enabled:
-        plan_name = f"{plan_name} - Report Builder (5 Reports)"
-    return plan_name
+    name_parts = [
+        product_rate.name if product is None else product.name,
+        (" Trial" if plan_key.is_trial else " Edition") if product is None else "",
+    ]
+    if plan_key.edition != SoftwarePlanEdition.ENTERPRISE:
```

Comment:
I think this change still isn't quite correct and going to think it through for a little. Rather than looking specifically for a plan being not `SoftwarePlanEdition.ENTERPRISE`, I probably should look if a plan is in `SoftwarePlanEdition.SELF_RENEWABLE_EDITIONS`, because it also wouldn't make sense to have a Community or Paused plan labeled as "Pay Monthly". Boy right now I wish our built-in software plans were just defined in code instead of as database objects.

Reply:
I'm in favor of checking if a plan is in `SoftwarePlanEdition.SELF_RENEWABLE_EDITIONS`